docs/docs/04-multiple-components.md

```diff
@@ -180,7 +180,7 @@ In React, data flows from owner to owned component through `props` as discussed
 
 ## A Note on Performance
 
-You may be thinking that it's expensive to react to changing data if there are a large number of nodes under an owner. The good news is that JavaScript is fast and `render()` methods tend to be quite simple, so in most applications this is extremely fast. Additionally, the bottleneck is almost always the DOM mutation and not JS execution and React will optimize this for you using batching and change detection.
+You may be thinking that it's expensive for React to change data if there are a large number of nodes under an owner. The good news is that JavaScript is fast and `render()` methods tend to be quite simple, so in most applications this is extremely fast. Additionally, the bottleneck is almost always the DOM mutation and not JS execution. React will optimize this for you using batching and change detection.
```

Comment:
I think I like this sentence as it was before. React isn't changing data (it is but that's not what is being discussed). The application is reacting to a change in the data it has (eg, props or state).